src/lib/keys.js

```diff
@@ -141,27 +142,33 @@ const deriveNetworkSeedFromMnemonic = async (
   return Just(await kg.deriveNetworkSeed(mnemonic, passphrase, revision - 1));
 };
 
+/**
+ * @param {string} authToken
+ * @param {number} number
+ * @return {Maybe<string>}
+ */
+export const deriveNetworkSeedFromAuthToken = (authToken, revision, point) => {
+  const networkSeed = shas(authToken, `revision-${point}-${revision}`)
+    .toString('hex')
+    .slice(0, 32);
+  return Just(networkSeed);
+};
+
 /**
  * @return {Promise<Maybe<string>>}
  */
 export const attemptNetworkSeedDerivation = async ({
   urbitWallet,
-  wallet,
-  authMnemonic,
-  details,
+  point,
+  authToken,
   revision,
 }) => {
   if (Just.hasInstance(urbitWallet)) {
     return await deriveNetworkSeedFromUrbitWallet(urbitWallet.value, revision);
   }
 
-  if (Just.hasInstance(wallet) && Just.hasInstance(authMnemonic)) {
-    return await deriveNetworkSeedFromManagementMnemonic(
```

Comment:
If we're logged in with the management mnemonic, we can still derive from that as per the wallet structure just fine. Is there a reason we shouldn't?